doc/classes/EditorSettings.xml

```diff
@@ -1292,6 +1292,9 @@
 		<member name="text_editor/completion/idle_parse_delay" type="float" setter="" getter="">
 			The delay in seconds after which the script editor should check for errors when the user stops typing.
 		</member>
+		<member name="text_editor/completion/idle_parse_delay_with_errors_found" type="float" setter="" getter="">
+			The delay used instead of [member text_editor/completion/idle_parse_delay], when the parser has found errors. A lower value should feel more responsive while fixing code.
```

Comment:
Depends on what do you mean by "responsive". Lower value means the script is parsed more often, which can result in delays.

Reply:
Then it may be worth mentioning something about processing cost here...